botocore/docs/example.py

```diff
@@ -15,6 +15,13 @@
 
 
 class BaseExampleDocumenter(object):
+    EVENT_NAME = ''
+
+    def __init__(self, service, operation, event_emitter):
+        self.service = service
```

Comment:
We've had an informal convention of not using the bare service/operation words and being more explicit about what about the service we're using: `service_name`, `service_model`, etc.  I'd prefer these to just be service_name/operation_name.